dat/autonav.lua

```diff
@@ -403,9 +403,27 @@ local function autonav_rampdown( count_brake )
    end
 end
 
+local function turnoff_afterburner()
+   for i,n in ipairs(player.pilot():actives()) do
+      -- Why does n:type() not work ?
+      -- Why *A*fterburner and not afterburner ?
+      -- The documentation is not clear : (too much or.s)
+      --   activate: boolean Whether or not to activate or deactivate
+      -- Proposition:
+      --   activate: boolean Whether to activate or deactivate
+      if n["type"]=="Afterburner" and n["state"]=="on" then
+         player.pilot():outfitToggle(i)
```

Comment:
```suggestion
         player.pilot():outfitToggle( n['slot'] )
```
`i` is not the slot index. `n['slot']` is.